_IBC/archway-omniflix.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "../ibc_data_schema.json",
+  "chain_1": {
+    "chain_name": "archway",
+    "client_id": "07-tendermint-39",
+    "connection_id": "connection-42"
+  },
+  "chain_2": {
+    "chain_name": "omniflix",
```

Comment:
chain name should be omniflixhub? -> https://github.com/cosmos/chain-registry/tree/master/omniflixhub

Reply:
Updated chain name to "omniflixhub".